google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerImplTest.java

```diff
@@ -70,4 +71,49 @@ public void createAndCloseSession() {
     // The same channelHint is passed for deleteSession (contained in "options").
     Mockito.verify(rpc).deleteSession(sessionName, options.getValue());
   }
+
+  @Test
+  public void getDbclientAgainGivesSame() {
```

Comment:
Is this part of the review ?

Reply:
Yes, this is the main part of the change, i.e. adding expanding test coverage.